src/plugins/data/public/filter_bar/filter_bar.tsx

```diff
@@ -25,12 +25,11 @@ import { METRIC_TYPE } from '@kbn/analytics';
 import { FilterEditor } from './filter_editor';
 import { FILTER_EDITOR_WIDTH, FilterItem } from './filter_item';
 import { FilterOptions } from './filter_options';
-import { useKibana } from '../../../../kibana_react/public';
-import { IDataPluginServices, IIndexPattern } from '../..';
+import { useKibana } from '../../../kibana_react/public';
```

Comment:
our thoughts were to keep the current API for the plugin date, since I see the related code `FilterLabel` `FilterItem` lies in the `esFilters` object.  If @ppisljar ok with moving it out of data plugin too, let's do it

Reply:
The unified plugin should have all the UI components that are part of it and FilterBar is def one of them.

Reply:
i am ok with moving FilterBar out, lets check if FilterLabel and FilterItem are ever imported from esFilters, else they should be removed from there.